src/prefs.ts

```diff
@@ -85,6 +85,33 @@ export default class GdeejPreferences extends ExtensionPreferences {
   }
 
   private _fillGeneralPage() {
+    {
+      const comboRowNoiseReduction = this._builder!.get_object(
+        'combo-row-noise-reduction'
+      ) as Adw.ComboRow;
+      
+      const comboRowOptions = new Gtk.StringList();
+      comboRowNoiseReduction.set_model(comboRowOptions);
+
+      const currentNoiseReduction = this._settings!.get_double(
+        settingsKeys.NOISE_REDUCTION
+      );
+      
+      comboRowOptions.append("10");
```

Comment:
not a fan of inline enums. Idk really where to move it tbh. Static field on `GdeejPreferences` probably? Something like

```js
static NOISE_REDUCTION_LEVELS = [10, 5, 3.5, 2.5, 1.5];
```

and then in `_fillGeneralPage`:

```js
for (const level of GdeejPreferences.NOISE_REDUCTION_LEVELS) {
  comboRowOptions.append(String(level));
}
```

not insisting on var names